runfrontendtests.sh

```diff
@@ -9,7 +9,7 @@ python -m tests.cypress.frontend_tests_initdb
 flask run -p 3002 --no-reload --debugger 2>&1 1>/tmp/funnel-server.log & echo $! > /tmp/funnel-server.pid
 function killserver() {
     kill $(cat /tmp/funnel-server.pid)
-    python -m tests.cypress.frontend_tests_dropdb
+    python -m frontend_tests_dropdb
```

Comment:
This won't work unless the file has moved to root folder, or the shell script has moved into the Cypress folder.

Reply:
In the next line, we cd into the cypress folder.

Reply:
Ah, we should `cd` out before calling this module then. This function itself has no context for where it is being called from, so the assumption is that it's still in the home folder.